lib/core/network/api_service.dart

```diff
@@ -80,7 +80,7 @@ class APIService {
       body: jsonEncode(parentGuid),
     );
 
-    log('fetch profiles status code: ${response.statusCode}');
+    log('fetch children status code: ${response.statusCode}');
```

Comment:
I would remove the debug logging from the code :)
It's getting logged to logit in dev and prod.
Also some other places below.

If its really helpful while debugging we could also make a general thing to log the API call + statuscode when in dev mode